delete.php

```diff
@@ -0,0 +1,16 @@
+<?php
+require_once 'src/Models/CarsModel.php';
+
+if (isset($_GET['delete']))
+{
+    $db = new PDO('mysql:host=db; dbname=Cars', 'root', 'password');
```

Comment:
It doesn't cause any issues, but you should set

`$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);`

on this $db before you pass it into the carsmodel

Good idea to always be consistent with it